samples/DataImporter/GraphEngine.DataImporter/DsvImporter.cs

```diff
@@ -8,15 +8,21 @@
 
 namespace GraphEngine.DataImporter
 {
-    class CsvImporter : IImporter<string>
+    class DsvImporter : IImporter<string>
```

Comment:
Why we rename this class? CsvImporter looks good to me even if the delimiters are not commas.

---
In reply to: [105551444](https://github.com/Microsoft/GraphEngine/pull/54#discussion_r105551444) [](ancestors = 105551444)